### DISCUSSION TITLE
Gray-out filtered messages

### DISCUSSION THREAD

**Discussion by Nerixyz** [Chatterino/chatterino2#4900] [Ideas]
I want to have messages from bots and commands for bots grayed-out, so messages from "normal" users stick out. I don't want to hide the messages completely.
Having two splits with one being filtered is not an option due to size constraints.

For me, it would be fine if the _Select filters_ dialog contained an option to gray-out messages.

**Comment by Felanbird** [comment]
mock-up:
![filters-gray-mockup-1](https://github.com/Chatterino/chatterino2/assets/41973452/5ccc24c4-7bd2-4fef-aea5-4f0c6c5f001a)

**Reply by Nerixyz** [reply]
Looks good, might need some space(r) to visually distinguish it from the actual filters.

**Comment by pajlada** [comment]
I'm open to this - it should be implemented as an enum rather than a bool in case we think of some other behaviour for filter display mode

**Comment by x9136** [comment]
I also propose to consider an option when messages, instead of being deleted, are replaced with a “Show” button. This way the chat would be cleaner, and it would be possible to view messages sometimes.

**Reply by pajlada** [reply]
Do you mean for messages that have been filtered out?

**Reply by x9136** [reply]
> Do you mean for messages that have been filtered out?

~Of course.~ Ah, I see. I consider messages that are hidden based on patterns as filtered. Perhaps I'll create a separate discussion.

**Comment by Mm2PL** [comment]
Accepted as issue #5217.
